legacy/emergency-freeze/20250604_084529/continuity-states/mcp-continuity-service/src/api/main.py

```
@@ -0,0 +1,213 @@
+        if current_user["subscription_plan"] != "enterprise":
+            if current_user["sessions_used"] >= current_user["sessions_limit"]:
+                raise HTTPException(
+                    status_code=403, 
+                    detail="Session limit reached. Please upgrade your subscription."
+                )
+        
+        result = await continuity_manager.process_user_input(
+            user_input=request.user_input,
+            session_id=request.session_id,
+            user_id=current_user["id"]
+        )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'user_id' in method call

(E1123)